### ISSUE TITLE
V2 feature  - orientation indicator in the top-right corner

### ISSUE THREAD

**Issue by hrgdavor** [jscad/OpenJSCAD.org#734]
## Expected Behavior
A way to know orientation when object covers XYZ axis lines

## Actual Behavior
no additional indicator

## Steps to Reproduce the Problem

  1. open web
  2. add a sphere
  3. try to guess where you are after rotating

## Specifications
example fusion360
![image](https://user-images.githubusercontent.com/2480762/99301816-c9046280-284e-11eb-99a6-1c1938bcfadb.png)
exmaple freecad
![image](https://user-images.githubusercontent.com/2480762/99301840-d6215180-284e-11eb-907c-37e93e62cf80.png)

I would be a nice thing to have, even without posibility to interact with it.

  - Version: V2
  - Platform: all
  - Environment: browser

**Comment by z3dev** [comment]
FYI, there are now keyboard shortcuts.

**Comment by hrgdavor** [comment]
Yes, I am aware of those. I am follwing th amailing list. I have already forgottne most of them, and I cant find in the help section the list of shortcuts :). Still, an indicator is very useful.

**Comment by hrgdavor** [comment]
posibly useful from stacko
https://stackoverflow.com/questions/60245700/three-js-rotate-group-with-same-orientation-as-camera-for-view-cube-orientation

**Comment by z3dev** [comment]
> Yes, I am aware of those. I am follwing th amailing list. I have already forgottne most of them, and I cant find in the help section the list of shortcuts :). Still, an indicator is very useful.

See the settings. The shortcuts can be changed.
